KS.Fiks.IO.Arkiv.Client/Schema/arkivmeldingOppdatering.xsd

```diff
@@ -0,0 +1,263 @@
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="korrespondansepartSlett">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+        </xs:sequence>
+    </xs:complexType>
+    
+    <xs:complexType name="merknadOppdateringer">
+        <xs:sequence>
+            <xs:element name="oppdatering" type="merknadOppdatering" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="slett" type="merknadSlett" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="ny" type="arkivmelding:merknad" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="merknadOppdatering">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+            <xs:element name="merknadstekst" type="n5mdk:merknadstekst" minOccurs="0"/>
+            <xs:element name="merknadstype" type="n5mdk:merknadstype" minOccurs="0"/>
+            <xs:element name="merknadsdato" type="n5mdk:merknadsdato" minOccurs="0"/>
+            <xs:element name="merknadRegistrertAv" type="n5mdk:merknadRegistrertAv" minOccurs="0"/>
+            <xs:element name="skjermetObjekt" type="n5mdk:skjermetObjekt" minOccurs="0"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="merknadSlett">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="klassifikasjonOppdateringer">
+        <xs:sequence>
+            <xs:element name="oppdatering" type="klassifikasjonOppdatering" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="slett" type="klassifikasjonSlett" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="ny" type="arkivmelding:klassifikasjon" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+    
+    <xs:complexType name="klassifikasjonOppdatering">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+            <xs:element name="klasseID" type="n5mdk:klasseID" minOccurs="0"/>
+            <xs:element name="tittel" type="n5mdk:tittel" minOccurs="0"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="klassifikasjonSlett">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="matrikkelnummerOppdateringer">
+        <xs:sequence>
+            <xs:element name="ny" type="arkivmelding:matrikkelnummer" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="slett" type="matrikkelnummerSlett" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="matrikkelnummerSlett"> <!-- her er vel alle til sammen nøkkelen for å slå opp. Eller burde det være mulig med en systemID her også? -->
```

Comment:
Hva er id her? Alle vil jeg anta?

Reply:
Det blir som for punkt. Men her er det heltall og ikke desimaltall. D.v.s. at det er enklere å oppdatere. Så da blir det alle...